doc/dev/corpus.rst

```diff
@@ -26,38 +26,28 @@ We can generate an object corpus for a particular version of ceph like so.
 	cd ceph
 	git submodule update --init --recursive
 
-#. Build with flag to dump objects to /tmp/foo::
-
-	rm -rf /tmp/foo ; mkdir /tmp/foo
-	./do_autogen.sh -e /tmp/foo
-	make
-
 #. Start via vstart::
 
-	cd src
-	MON=3 OSD=3 MDS=3 RGW=1 ./vstart.sh -n -x
+	cd build
+	MON=3 OSD=3 MDS=3 RGW=1 ../src/vstart.sh -n -x
 
 #. Use as much functionality of the cluster as you can, to exercise as many object encoder methods as possible::
 
-	./rados -p rbd bench 10 write -b 123
-	./ceph osd out 0
-	./init-ceph stop osd.1
-	for f in ../qa/workunits/cls/*.sh ; do PATH=".:$PATH" $f ; done
-	../qa/workunits/rados/test.sh
-	./ceph_test_librbd
-	./ceph_test_libcephfs
-	./init-ceph restart mds.a
+	bin/ceph osd pool create mypool 8
+	bin/rados -p rbd mypool 10 write -b 123
+	bin/ceph osd out 0
+	bin/init-ceph stop osd.1
+	for f in ../qa/workunits/cls/*.sh ; do PATH="bin:$PATH" $f ; done
+	PATH="bin:$PATH" ../qa/workunits/rados/test.sh
```

Comment:
We can probably elliminate these modifications to "PATH" as well with the command I suggested above.